shared/teams/team/members/container.js

```diff
@@ -12,54 +12,55 @@ export type OwnProps = {
 const order = {owner: 0, admin: 1, writer: 2, reader: 3}
 
 const getOrderedMemberArray = (
-  memberInfo: I.Set<Types.MemberInfo>,
+  memberInfo: I.Map<string, Types.MemberInfo>,
   you: ?string,
   listYouFirst: boolean
 ): Array<Types.MemberInfo> => {
-  let info = memberInfo
-  let returnArray = info.toArray().sort((a, b) => {
-    if (!a.active) {
-      if (!b.active) {
-        // both are inactive, compare usernames
-        return a.username.localeCompare(b.username)
-      }
-      // b is active, should go later
-      return -1
-    } else if (!b.active) {
-      // b is inactive, should come first
-      return 1
-    }
-    if (listYouFirst && you) {
-      if (a.username === you) {
+  let returnArray = I.Set(memberInfo.values())
```

Comment:
:sheep: this is a little confusing, in other places we do `memberInfo.valueSeq().toArray()` (admittedly not  much clearer)

Reply:
Done!